@here/harp-mapview/lib/geometry/TileGeometryManager.ts

```diff
@@ -36,8 +154,154 @@ export abstract class TileGeometryManagerBase implements TileGeometryManager {
     abstract initTile(tile: Tile): void;
 
     abstract updateTiles(tiles: Tile[]): void;
-}
 
+    clear(): void {
+        this.enabledKinds = new GeometryKindSet();
```

Comment:
just clear sets instead of creating new objects?

Reply:
Done